src/Middleware/ExceptionHandlingMiddleware.php

```diff
@@ -31,6 +31,7 @@ public function process(
         RequestHandlerInterface $handler
     ) : ResponseInterface {
         try {
+            $status = 200;
```

Comment:
@driusan I don't think this does anything since next line returns.

Reply:
It fixes the `src/Middleware/ExceptionHandlingMiddleware.php:51 PhanPossiblyUndeclaredVariable Variable $status is possibly undeclared` error